app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -189,8 +195,22 @@ class ConversationViewModel @Inject constructor(
         onDialogDismissed()
     }
 
-    private fun List<com.wire.kalium.logic.data.message.Message>.toUIMessages(): List<Message> {
+    private fun List<MemberDetails>.findSender(senderId: UserId): MemberDetails? = firstOrNull { member ->
+        when (member) {
+            is MemberDetails.Other -> member.otherUser.id == senderId
+            is MemberDetails.Self -> member.selfUser.id == senderId
+        }
+    }
+
+    private val MemberDetails.name
+        get() = when (this) {
+            is MemberDetails.Other -> this.otherUser.name
+            is MemberDetails.Self -> this.selfUser.name
+        }
+
+    private fun List<com.wire.kalium.logic.data.message.Message>.toUIMessages(members: List<MemberDetails>): List<Message> {
```

Comment:
Do we need to have the import here?

Reply:
There are other usages of `com.wire.android.ui.home.conversations.model.Message` in this file.

Maybe an import alias? But this class is **only** being used inside this function (in this file).
I am not a huge fan of import aliases, especially for a single usage.

Reply:
What about renaming that `com.wire.android.ui.home.conversations.model.Message` to `MessageView` or `ConversationMessage` ?